osu.Game.Modes.Mania/UI/ManiaHitRenderer.cs

```diff
@@ -12,7 +13,8 @@ public class ManiaHitRenderer : HitRenderer<ManiaBaseHit>
     {
         private readonly int columns;
 
-        public ManiaHitRenderer(int columns = 5)
+        public ManiaHitRenderer(Beatmap beatmap, int columns = 5)

```

Comment:
i feel like column count won't be required as it should be part of the beatmap data here. but we can address this when mania is actually implemented. (although it does bring the question to mind of whether we should be passing in a `WorkingBeatmap` instead of just `Beatmap`, so we have mods etc.).